DS/DSAssignment2/Stack.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Scanner;
+
+
+public class Stack {
+private int top;
+private int size;
+Node<Object> node;
+
+
+
+public int getTop() {
+	return top;
+}
+
+public void setTop(int top) {
+	this.top = top;
+}
+
+public int getSize() {
+	return size;
+}
+
+public void setSize(int size) {
+	this.size = size;
+}
+
+public Node<Object> getNode() {
+	return node;
+}
+
+public void setNode(Node<Object> node) {
+	this.node = node;
+}
+
+public Stack(int size)
+{
+   
+	this.size=size;
+	top=-1;
+	node =new Node<Object>();
+	
+}
+
+public void push(Object item)
+{
+	
+	if(top>=size-1)
+	{
+		System.out.println("Stack overflow");
+		
+	}else
+	{
+		top=top+1;
+		node.add(item);
+		System.out.println(top);
+		
+	}
+
+}
+
+public void pop()
```

Comment:
ACTION: Pop must return the value which was popped as the name of the method suggests